evaluation.md

```diff
@@ -5,6 +5,27 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+We got into the IDE and immediately realized that things were going to be harder than we thought they would be. We  went through various discussions of what would be objects,classes, and operators and eventually settled on a compromise of where to start. We changed to:
+
+'rule( # seq ) -> action' which is pretty different but we're planning to slim down significantly by removing the need for the rule keyword.
```

Comment:
Yeah, by the time we finally managed to get the syntax to work with the keyword, we were pretty exhausted (and a little frustrated) from dealing with Scala and making tradeoffs.